2_Operations/investment-strategy.md

```diff
@@ -0,0 +1,30 @@
+# Investment Strategy
+
+Niteo keeps reserves to [cover close to 12 months of average expenses](/5_People/profit-sharing.md). To not lose this money to inflation, we invest it into appreciating assets.
+
+
+## Short-Term Pool
+
+Reserves in safe and liquid assets. We try to keep the currency equally split between EUR and USD. Around 30% of total reserves.
+
+Assets:
+- cash on bank accounts
+- bonds (15% of total) with Vanguard BNDW (total world bonds in USD)
+
+
+## Long-Term Pool
+
+Reserves in more risky or illiquid assets. The currency split is not relevant because most of these assets compensate for the difference between EUR and USD. Around 70% of total reserves.
```

Comment:
```suggestion
Reserves in more risky or illiquid assets. The currency split is irrelevant because most of these assets are global investments. Up to 70% of all reserves can be in this pool.
```